inbm-lib/inbm_common_lib/utility.py

```diff
@@ -271,3 +271,21 @@ def validate_file_type(path: list[str]) -> None:
     if os.path.exists(TEMP_EXT_FOLDER):
         shutil.rmtree(TEMP_EXT_FOLDER, ignore_errors=True)
     remove_file_list(extracted_file_list)
+
+
+def get_os_version() -> str:
+    """Get os version from os release file.
+
+    @return value of the VERSION
+    """
+    if os.path.exists(OS_RELEASE_PATH):
+        with open(OS_RELEASE_PATH, 'r') as version_file:
+            for line in version_file:
+                if line.startswith('VERSION='):
+                    version = line.split('=')[1].replace('\n', ' ')
```

Comment:
This will not handle quotes properly. E.g. if you run it on a file with `VERSION="abc def"`, it will return: `"abc def"` rather than `abc def`

I would suggest making a function that takes in a string corresponding to the contents of the file, and returns a Dict[str, str]. I ran this through the AI and I got this as a suggestion:

```
from typing import Dict, Tuple, Optional
import ast

def parse_os_release(file_content: str) -> Dict[str, str]:
    """
    Parses the content of an os-release file and returns a dictionary of key-value pairs.

    :param file_content: The content of the os-release file as a string.
    :return: A dictionary containing key-value pairs from the file.
    """
    result: Dict[str, str] = {}
    for line in file_content.splitlines():
        parsed = parse_line(line)
        if parsed:
            key, value = parsed
            result[key] = value
    return result

def parse_line(line: str) -> Optional[Tuple[str, str]]:
    """
    Parses a line of the os-release file and returns a key-value pair.
    Returns None if the line is empty or a comment.

    :param line: A line from the os-release file.
    :return: A tuple of (key, value) if the line contains a key-value pair, else None.
    """
    line = line.strip()
    if not line or line.startswith('#'):
        return None
    if '=' not in line:
        return None
    key, value = line.split('=', 1)
    key = key.strip()
    value = value.strip()
    if (value.startswith('"') and value.endswith('"')) or \
       (value.startswith("'") and value.endswith("'")):
        try:
            value = ast.literal_eval(value)
        except (SyntaxError, ValueError):
            pass  # Keep value as is
    # Else, value is unquoted, keep as is
    return key, value
```

Reply:
Added this method.